harness/determined/cli/render.py

```diff
@@ -1,3 +1,15 @@
+"""Rendering utilities for the CLI.
+
+The to_json functions in this module are used to convert resource objects into dicts that look
+similar to their analagous resources from bindings to_json functions.
+
+This allows the CLI to use the similar code to render resources from both the bindings and the
+experimental API.
+
+For example, the following two lines of pseudocode return similar objects:
+* bindings.v1Model.to_json()
+* _render.model_to_json(model.Model)
+"""
```

Comment:
"""This module provides CLI rendering utilities.

The to_json functions in this module are used to convert resource objects into dictionaries that look similar to their analogous resources from
bindings' to_json functions. 

This consistency allows the CLI to render resources
from both bindings and the experimental API in a similar manner.

Example:
Similar objects are obtained from the following pseudocode:
* bindings.v1Model.to_json()
* _render.model_to_json(model.Model)